code/datums/station_traits/negative_traits.dm

```diff
@@ -100,7 +100,7 @@
 /datum/station_trait/overflow_job_bureaucracy
 	name = "Overflow bureaucracy mistake"
 	trait_type = STATION_TRAIT_NEGATIVE
-	weight = 0 //SKYRAT EDIT: - CHANGES WEIGHT FROM FIVE TO ZERO
+	weight = 2 //BUBBERS EDIT: FUCK SKYRAT WE WANT BUREAUCRACY, OLD SKYRAT WEIGHT: 0
```

Comment:
Ehh.... sure I guess. Should perhaps be 1 though, dunno

Reply:
Sure, 1 can work.